packages/jest-jasmine2/src/jasmine/Suite.js

```diff
@@ -190,4 +190,12 @@ function isFailure(args) {
   return !args[0];
 }
 
+function isFunction(fn, fnName) {
+  if (typeof fn.fn === 'string') {
+    throw new Error(`First argument in the ${fnName} hook cant't be a string`);
```

Comment:
What if the first argument is a number?
We'd rather like to test if `fn` is actually a function.